travel-event-host-app/src/components/footer/footer.module.css

```diff
@@ -0,0 +1,20 @@
+@import url('https://fonts.googleapis.com/css2?family=Montserrat&family=Roboto:wght@400;500&display=swap');
```

Comment:
@davideastmond Already import the Roboto family on layout so all the project have Roboto as default, if you will use the Montserrat family use the font module, font module import the weights dynamically

```
import { Montserrat } from 'next/font/google'
const font= Montserrat({ subsets: ['latin'] })

export default function Footer() {
  return (
   <div className={`${styles.footer} ${font.Montserrat}`}>
            <a>About us</a>
            <a>Chingu</a>
            <a>Team</a>
        </div>
  )
}

```

Reply:
I just fixed the style use now.